interfaces.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"k8s.io/client-go/tools/record"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+)
+
+type eventRecorder interface {
```

Comment:
🚫 **[checkstyle]** <unused> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>type `eventRecorder` is unused